jennifer/app/src/views/home/index.ecr.lqd

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div id="logo"></div>
+  <div class="col-sm-12 col-md-6">
+    <h2><%= t "welcome_to_amber" %></h2>
+    <p>Thank you for trying out the Amber Framework.  We are working hard to provide a super fast and reliable framework that provides all the productivity tools you are used to without sacrificing the speed.</p>
+    <div class="list-group">
+      <a class="list-group-item list-group-item-action" target="_blank" href="https://docs.amberframework.org">Getting Started with Amber Framework</a>
+      <a class="list-group-item list-group-item-action" target="_blank" href="https://github.com/imdrasil/jennifer.cr/blob/master/docs/index.md">Getting Started with Jennifer ORM</a>
+      <a class="list-group-item list-group-item-action" target="_blank" href="https://github.com/veelenga/awesome-crystal">List of Awesome Crystal projects and shards</a>
+      <a class="list-group-item list-group-item-action" target="_blank" href="https://crystalshards.xyz">What's hot in Crystal right now</a>
+    </div>
+  </div>
+</div>
```

Comment:
New Line

Reply:
Should I add a newline in similar places for other recipes?

Reply:
We should do that in a separate PR